packages/react-instantsearch-core/src/connectors/connectToggleRefinement.js

```diff
@@ -24,10 +26,11 @@ function getCurrentRefinement(props, searchState, context) {
     false
   );
 
-  if (currentRefinement) {
-    return currentRefinement;
+  if (falsyStrings.indexOf(currentRefinement) !== -1) {
```

Comment:
We could have wrote the condition on the `'true'` that would include all falsy values.

```js
if (typeof currentRefinement === 'string') {
  return currentRefinement === 'true';
}
```